step.yml

```diff
@@ -0,0 +1,172 @@
+---
+title: Create App Store Release
+summary: The Step creates an App Store release with the Release Management feature.
+description: |-
+  Create and configure a new release to App Store Connect with Release Management.
+website: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release
+support_url: https://github.com/bitrise-steplib/bitrise-step-create-new-app-store-release/issues
+
+type_tags:
+- deploy
+
+project_type_tags:
+- ios
+- macos
```

Comment:
Do we really support macOS deployment? The rest of the step.yml seems to be about ios.